### DISCUSSION TITLE
using wildcard for workspace names to forward output

### DISCUSSION THREAD

**Discussion by benibilme** [i3/i3#5881] [Q&A] [answered]
Hello,

I am using workspace numbers between 11-99 similar to  
[dopamine](git@github-benibilme:/EllaTheCat/dopamine-2020.git)

I would like output 1-49 my left monitor and the rest to my right monitor.  Is there a easy way to do this. Do I need to specifically type `workspace 33 output $left`  for each workspace. Can  I use a wildcard or regular expression pattern in i3wm configuration?

**Comment by orestisfl** [comment, marked as answer]
Yes, you need to configure each workspace individually. You can write a quick script to generate the config once. Otherwise, you can use the [IPC interface](https://i3wm.org/docs/ipc.html) to dynamically decide which workspace should go to which output but I feel it would be overkill for your case.
